components/formats-gpl/src/loci/formats/in/CellWorxReader.java

```diff
@@ -711,14 +714,16 @@ private IFormatReader getReader(String file)
       pnl = new MetamorphReader();
     }
 
-    IMetadata metadata;
-    try{
-      metadata = service.createOMEXMLMetadata();
-    }
-    catch (ServiceException exc) {
-      throw new FormatException("Could not create OME-XML store.", exc);
+    if (omexml) {
+      IMetadata metadata;
+      try {
```

Comment:
Now that OMEXMLServiceImpl is in formats-api, are there any circumstances in which createOMEXMLMetadata could throw?  I'm wondering if all use of the OMEXMLService API could be simplified if it would be possible to remove the `throws ServiceException` from their methods.